pkg/test/resourcefixture/testdata/basic/firestore/v1beta1/firestoredatabase/firestoredatabase-minimal/_http.log

```diff
@@ -171,18 +171,16 @@ X-Xss-Protection: 0
   },
   "name": "projects/${projectId}/databases/firestoredatabase-${uniqueId}/operations/${operationID}",
   "response": {
-    "@type": "type.googleapis.com/google.firestore.admin.v1.Database",
+    "@type": "type.googleapis.com/mockgcp.firestore.admin.v1.Database",
     "appEngineIntegrationMode": 2,
     "concurrencyMode": 2,
     "createTime": "2024-04-01T12:34:56.123456Z",
     "deleteProtectionState": 1,
-    "deleteTime": "2024-04-01T12:34:56.123456Z",
     "earliestVersionTime": "2024-04-01T12:34:56.123456Z",
     "etag": "abcdef0123A=",
     "locationId": "us-west2",
     "name": "projects/${projectId}/databases/firestoredatabase-${uniqueId}",
     "pointInTimeRecoveryEnablement": 2,
-    "previousId": "firestoredatabase-${uniqueId}",
```

Comment:
super nit: propagate this field in DELETE resposne.

Reply:
This is a weird quirk of the API. This field is not defined in the public proto. I am not sure how to set it (?) Seems like an internal-only proto that is returning this field. Same with `deleteTime`.